logic/src/commonMain/kotlin/com/wire/kalium/logic/data/user/UserRepository.kt

```diff
@@ -56,6 +56,7 @@ interface UserRepository {
     suspend fun updateLocalSelfUserHandle(handle: String)
     suspend fun getAllKnownUsers(): Either<StorageFailure, List<OtherUser>>
     suspend fun getKnownUser(userId: UserId): Flow<OtherUser?>
+    fun getKnownUserMinimized(userId: UserId): OtherUserMinimized?
```

Comment:
```
    suspend fun getKnownUserMinimized(userId: UserId): OtherUserMinimized?
```

The idea of making it suspendable is that even though SQLDelight queries are not suspendable for us yet, they can be in the near future, as [Async drivers are being supported](https://github.com/cashapp/sqldelight/commit/371f5038d03090b65e10bc79dd190973849e863c) and we will need them for JS.

This way we don't need to break compatibility in the future.

Reply:
okay I will update it